tests/scripts/depends.py

```diff
@@ -512,8 +512,23 @@ def __init__(self, options, conf):
 
         # Find hash modules by name.
         hash_symbols = self.config_symbols_matching(r'MBEDTLS_(MD|RIPEMD|SHA)[0-9]+_C\Z')
-        # Find elliptic curve enabling macros by name.
-        curve_symbols = self.config_symbols_matching(r'MBEDTLS_ECP_DP_\w+_ENABLED\Z')
+
+        # Find elliptic curve enabling macros
+        # Mapping is needed for PSA_WANT_ECC_SECP_K1_224 because it actually uses 225 bits.
+        key_type_mapping = {('PSA_ECC_FAMILY_SECP_K1', '225'): ('PSA_ECC_FAMILY_SECP_K1', '224')}
+        def get_symbol_from_key_type(key_type_family, bit_size):
+            (family_name, corrected_bit_size) = key_type_mapping.get((key_type_family, bit_size),
+                                                                     (key_type_family, bit_size))
+            symbol = psa_information.finish_family_dependency(family_name, corrected_bit_size)
+            return psa_information.psa_want_symbol(symbol)
+
+        curve_symbols = {symbol
```

Comment:
Another thought, for the `cipher` domain that kind of code based on `psa_info` was necessary but for curves it seems that just something like `curve_symbols = self.config_symbols_matching(r'PSA_WANT_ECC_')` would be enough.